lesson4/Homework/timer.js

```diff
@@ -0,0 +1,107 @@
+// Time values
+hours = "00";
```

Comment:
General advice. Do not declare global variables. Treat hours, minutes and seconds as numerical data especially if you're doing arithmetics with them and only convert them to string in display function. Also always use strict comparison `===`. Try to use variable names as descriptive as possible. For example `valArr` could be named as `timeValues`. And instead of array it could be an object `{hours: 0, minutes: 0, seconds: 10}` so it will be even more informative.